src/Hal/Metric/System/Packages/Composer/Packagist.php

```diff
@@ -1,6 +1,8 @@
 <?php
 namespace Hal\Metric\System\Packages\Composer;
 
+use function is_array;
```

Comment:
Please avoid too recent code. This syntax is not compatible witll all supported PHP version